src/components/tooltip/tooltip.js

```diff
@@ -311,6 +312,7 @@ function MdTooltipDirective($timeout, $window, $$rAF, $document, $interpolate,
       }
 
       function onElementDestroy() {
+        panelRef && panelRef.destroy();
```

Comment:
L300 occurs when the tooltip's scope is destroyed. L315 occurs when the parent element's scope is destroyed.